### PR TITLE
Feat/mid server

### PR DESCRIPTION
**Pull request by ivyhu630**
Updated index.js to handle all get and post request with instructions for use case.
Added sampleData in client/src/sampleData.js

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1 +1,49 @@
...
+app.use(express.static(path.join(__dirname, '../client/dist')));
+app.use(express.json());
+
+// Set up get method and post method
+// route will handle all request and routes no matter destination
+// This might need to be refactored if we want to handle different destinations
+// in unique ways but good for the purpose of the current middle server
+
+// Get products use case:
+// Note: ******might not need to pre-pend localhost. Let us know.*******
```
Comment (ryand8008) [inline server/index.js:19]: I'll be sure to take note of this when I start building out the widget

File: client/src/sampleData.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
+const sampleData = {
+  products: [
```
Comment (ryand8008) [inline client/src/sampleData.js:2]: Can we add sample data of the product information and product styles as well?

**Review by ryand8008 — APPROVED** [review]

File: example.env [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Copy the contents of this file into a new file called .env.
+
+# The following key-value pairs will be added to process.env
+
+GITHUB_APIKEY ="input_your_peronal_API_Key"
```
Comment (floridamaniac) [inline example.env:5]: Thank you for including this!

File: server/index.js [hunk 4]
```diff
@@ -1 +1,49 @@
...
+  const url = `https://app-hrsei-api.herokuapp.com/api/fec2/rfp${req.url}`;
+  axios({
+    url,
+    method,
+    params,
+    data,
+    headers: { Authorization: `${process.env.GITHUB_APIKEY}` },
+  })
+    .then((result) => {
+      const newData = result.data;
```
Comment (floridamaniac) [inline server/index.js:42]: Does this also send back the status code received from the api request? Like, if we send a bad request, does this always send back a `200`?

Comment (floridamaniac) [reply server/index.js:42]: Tested it in postman, looks like that would be captured in the error catch. Can we return that error response to the client in addition to logging it in the console?

**Review by floridamaniac — CHANGES_REQUESTED** [review]
Looks good! Only note on my end would be to still pass along the error responses so that the request doesn't hang if it's not a valid one.

File: client/src/sampleData.js (hunk 2 above)
Comment (ivyhu630) [reply client/src/sampleData.js:2]: Done

File: server/index.js (hunk 4 above)
Comment (ivyhu630) [reply server/index.js:42]: This has been fixed. Err will be sent back to client as well as a 404 status code

Comment (floridamaniac) [reply server/index.js:42]: Awesome, works on my end!

**Review by floridamaniac — APPROVED** [review]

**Pull request merged**